src/pokete/classes/fight/fight.py

```
@@ -150,6 +172,15 @@ def __call__(self, ctx: Context, providers: list[Provider]):
             self.fightmap.show()
             time.sleep(SPEED_OF_TIME * 0.5)
 
+            # Capture Mutator Impact before death check
+            for prov in self.providers:
+                if not prov.curr.player: # Enemy anomalies
+                    impact = (prov.curr.oldhp - prov.curr.hp) if hasattr(prov.curr, 'oldhp') else 0
+                    if getattr(prov.curr, 'is_boss', False): mutator_impact["BOSS"] += impact
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)